ups-connector/processes/AddressValidation.p.json

```diff
@@ -0,0 +1,87 @@
+{
+  "format" : "10.0.0",
+  "id" : "18B94088D7BAEC17",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "ups.connector.AddressValidationData"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "addressValidation(Integer,String,String,Integer,XAVRequest)",
+      "config" : {
+        "callSignature" : "addressValidation",
+        "input" : {
+          "params" : [
+            { "name" : "requestoption", "type" : "Integer" },
```

Comment:
Hi @ny-huynh , could you please change the names of some variables that weren't good? For example, change 'requestoption' to 'requestOption'.
Please make similar changes for the remaining variables.